purchase_archive/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
Why redefine the security groups, as they are never used in this module?

Reply:
Because I'm removing the deletion right as now they are archived instead.